src/main/java/com/nextstep/lotto/view/LottoMain.java

```diff
@@ -0,0 +1,19 @@
+package com.nextstep.lotto.view;
+
+import com.nextstep.lotto.domain.LottoSeller;
+import com.nextstep.lotto.domain.Lottos;
+
+public class LottoMain {
+    public static void main(String[] args) {
+        new LottoMain().run();
+    }
+
+    public void run() {
```

Comment:
```suggestion
    private static void run() {
```
이렇게 바꿔도 되지 않을까요?
객체 자신을 또 만들어서 run 메서드를 호출해야하는 이유가 무엇일까요?